LICENSE

```diff
@@ -1,3 +1,5 @@
+Copyright (C) 2015 Button, Inc
```

Comment:
I did 2014-2016 as that's what I see in other license files (although for android it would be 2015-2016)